docs/docs/faq.md

```diff
@@ -0,0 +1,63 @@
+---
+slug: /faq/
+sidebar_position: 4
+---
+
+# FAQ
+
+Thiws page contains a list of frequently asked questions about Task.
+
+- [Why should I use Task over ..?](#why-should-i-use-task-over-)
+- [Why won't my task update my shell environment?](#why-wont-my-task-update-my-shell-environment)
+- ['x' builtin command doesn't work on Windows](#x-builtin-command-doesnt-work-on-windows)
+
+## Why should I use Task over ..?
+
+WIP
+
+### Vs. Make
+
+WIP
+
+## Why won't my task update my shell environment?
+
+This is a limitation of how shells work. Task runs as a subprocess of your
+current shell, so it can't change the environment of the shell that started it.
+This limitation is shared by other task runners and build tools too.
+
+A common way to work around this is to create a task that will generate output
+that can be parsed by your shell. For example, to set an environment variable on
+your shell you can write a task like this:
+
+```yaml
+my-shell-env:
+  cmds:
+    - echo "export FOO=foo"
+    - echo "export BAR=bar"
+```
+
+Now run `eval $(task my-shell-env)` and the variables `$FOO` and `$BAR` will be
+available in your shell.
+
+## 'x' builtin command doesn't work on Windows
+
+The default shell on Windows (`cmd` and `powershell`) do not have commands like
```

Comment:
Just a FYI, it seems to PowerShell _do_ allow some commands like `cp` but they're actually aliases to other commands with a different interface, so far from being 100% compatible.

- https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/copy-item?view=powershell-7.3
- https://ilovepowershell.com/2011/11/03/list-of-top-powershell-alias/